services/offline/src/lib/dhis2-connection-status/dhis2-connection-status.tsx

```diff
@@ -183,23 +193,23 @@ export const Dhis2ConnectionStatusProvider = ({
     }, [offlineInterface, onUpdate])
 
     // Memoize this value to prevent unnecessary rerenders of context provider
-    const contextValue = useMemo(
-        () => ({
-            // in the unlikely circumstance that offlineInterface.latestIsConnected
-            // is `null` when this initializes, fail safe by defaulting to
-            // `isConnected: false`
-            isConnected: Boolean(isConnected),
-            isDisconnected: !isConnected,
-            lastConnected: isConnected
+    const contextValue = useMemo(() => {
+        // in the unlikely circumstance that offlineInterface.latestIsConnected
+        // is `null` or `undefined` when this initializes, fail safe by defaulting to
+        // `isConnected: true`. A ping or SW update should update the status shortly.
+        const validatedIsConnected = isConnected ?? true
+        return {
+            isConnected: validatedIsConnected,
+            isDisconnected: !validatedIsConnected,
+            lastConnected: validatedIsConnected
```

Comment:
This is a small fix: the correct fail-safe should be to default to `isConnected: true` so the app can be accessed